lib/components/app/batch-routing-panel.js

```diff
@@ -68,8 +71,7 @@ class BatchRoutingPanel extends Component {
         <div
           className="desktop-narrative-container"
           style={{
-            flexGrow: 1,
-            overflowY: 'hidden'
+            flexGrow: 1
           }}
```

Comment:
Remove all of it. (allowing scrolling removes any bounds that would apply for a flex layout)

Reply:
Also, around Line 33, remove `display: 'flex'` and `flexDirection` from the `ViewerContainer` style.

Reply:
Removing this causes the background of the pane to not extend to the full window height.